### ISSUE TITLE
best practice to add a stream config?

### ISSUE THREAD

**Issue by udos** [nginx/docker-nginx#953]
hi,

I would like to add a stream config similiar to this:
```
stream {
    server {
      listen 1234;
    
      proxy_connect_timeout 60s;
      proxy_socket_keepalive on;
      proxy_pass localhost:5432;
    }
}
```

how can I do this?

_note: I can not add it to /etc/nginx/conf.d because that is the include directory for http configs..._

**Comment by Kornelius777** [comment]
at the end of `nginx.conf`, I added:
```
stream {
include /etc/nginx/streams/*;
}
```

then, I added that directory:
(From within the container:)
`mkdir -p /etc/nginx/streams/`

now, just place your `stream { ... }` in a `stream.conf` inside this very directory.

**Comment by thresheek** [comment]
Indeed, it's best to provide an edited `/etc/nginx/nginx.conf` which will include the directory where you'd want to put the stream configuration files to.
